go/vt/tabletserver/schema_info.go

```diff
@@ -491,12 +491,12 @@ func (si *SchemaInfo) GetTable(tableName sqlparser.TableIdent) *TableInfo {
 }
 
 // GetSchema returns a copy of the schema.
-func (si *SchemaInfo) GetSchema() []*schema.Table {
+func (si *SchemaInfo) GetSchema() map[string]*schema.Table {
```

Comment:
It's not really a copy since the pointers point to the original objects? Probably needs to be mentioned in the comment, *schema.Table need to be considered RO.